src/nbl/video/utilities/CAssetConverter.cpp

```diff
@@ -955,6 +1093,50 @@ bool CAssetConverter::CHashCache::hash_impl::operator()(lookup_t<ICPUBuffer> loo
 	hasher.update(&patchedParams,sizeof(patchedParams)) << lookup.asset->getContentHash();
 	return true;
 }
+bool CAssetConverter::CHashCache::hash_impl::operator()(lookup_t<ICPUBottomLevelAccelerationStructure> lookup)
+{
+	// extras from the patch
+	hasher << lookup.patch->hostBuild;
+	hasher << lookup.patch->compactAfterBuild;
+	// overriden flags
+	hasher << lookup.patch->getBuildFlags(lookup.asset);
+	// finally the contents
+//TODO:	hasher << lookup.asset->getContentHash();
```

Comment:
note to self, need to make the `ICPUBottomLevelAccelerationStructure` and `IPreHashed`